mnefun/_mnefun.py

```diff
@@ -2341,7 +2341,11 @@ def do_preprocessing_combined(p, subjects, run_indices):
                             reject_tmin=p.reject_tmin,
                             reject_tmax=p.reject_tmax)
             # channel scores from drop log
-            scores = Counter([ch for d in epochs.drop_log for ch in d])
+            drops = Counter([ch for d in epochs.drop_log for ch in d])
+            # get rid of non-channel reasons in drop log
+            reasons = ['IGNORED', 'NO_DATA', 'USER', 'TOO_SHORT']
+            scores = {kk: vv for kk, vv in drops.items() if
+                      kk not in reasons}
```

Comment:
The more future-compatible check (in case MNE adds new reasons) would be to check `if kk in epochs.ch_names` rather than `if kk not in reasons`